app/components/shared/Panel/index.js

```diff
@@ -39,9 +40,9 @@ Panel.IntroWithButton = IntroWithButton;
 Panel.Row = Row;
 Panel.RowActions = RowActions;
 Panel.RowLink = RowLink;
+Panel.Header = Header;
 
 const SIMPLE_COMPONENTS = {
-  Header: 'bg-silver py2 px3 semi-bold',
```

Comment:
I’d really like to make it so such element changes don’t require us to be splitting these back out into files like this. I was thinking about this today while I was doing #230 :(